scipy/optimize/_linprog.py

```diff
@@ -165,67 +165,83 @@ def linprog(c, A_ub=None, b_ub=None, A_eq=None, b_eq=None,
             bounds=None, method='simplex', callback=None,
             options=None, x0=None):
     """
-    Minimize a linear objective function subject to linear
-    equality and inequality constraints. Linear Programming is intended to
-    solve the following problem form:
+    Linear programming: minimize a linear objective function subject to linear
+    equality and inequality constraints.
 
-    Minimize::
+    Linear programming solves problems of the following form:
+
+    .. math::
+
+        \min_x \ & c^T x \\\\
+        \mbox{such that} \ & A_{ub} x \leq b_{ub},\\\\
+        & A_{eq} x = b_{eq},\\\\
+        & l \leq x \leq u ,
+
+    where :math:`x` is a vector of decision variables; :math:`c`,
+    :math:`b_{ub}`, :math:`b_{eq}`, :math:`l`, and :math:`u` are vectors; and
+    :math:`A_{ub}` and :math:`A_{eq}` are matrices.
+
+    Informally, that's:
```

Comment:
In light of the new mathematical notation, I don't think the below explanation is needed except for stating the values of the default upper and lower bounds.

Reply:
We have to keep in mind that it's hard to read in the terminal between all those `math:` and backticks. Most of the time, that's the reason for hybrid notation.

Reply:
@ilayn thanks. Does that mean the math notation is OK but leave the hybrid, or get rid of the math notation?

Reply:
Just to reminder not to over TeXify the docs since most of the cases you would be looking at the docs on a terminal or any other docstring printing medium where TeX is not rendered.